spec/features/download_layer_spec.rb

```diff
@@ -12,13 +12,13 @@
   end
 
   scenario "clicking initial shapefile download button should trigger download", js: true do
-    expect(shapefile_download).to receive(:get).and_return("mit-f6rqs4ucovjk2-shapefile.zip")
-    visit solr_document_path("mit-f6rqs4ucovjk2")
+    expect(shapefile_download).to receive(:get).and_return("tufts-cambridgegrid100-04-shapefile.zip")
+    visit solr_document_path("tufts-cambridgegrid100-04")
     find("#downloads-button").click
     find('a[data-download-type="shapefile"]', text: "Export Shapefile").click
     expect(page).to have_css(
-      'a[href="/download/file/mit-f6rqs4ucovjk2-shapefile.zip"]',
-      text: "Your file mit-f6rqs4ucovjk2-shapefile.zip is ready for download"
+      'a[href="/download/file/tufts-cambridgegrid100-04-shapefile.zip"]',
+      text: "Your file tufts-cambridgegrid100-04-shapefile.zip is ready for download"
```

Comment:
this tufts-cambridge fixture actually has files available to download in the fixtures, and since we auto-click to download, we should use fixtures where this works